File: app/src/main/java/com/zhihudailytest/Utils/SPUtils.java
package com.zhihudailytest.Utils;

import android.content.Context;
import android.content.SharedPreferences;

import com.zhihudailytest.MyApplication;

/**
 * Created by Administrator on 2016/8/2.
 */
public class SPUtils {
    public static SharedPreferences.Editor getEditor(){
        SharedPreferences.Editor editor= MyApplication.ApplicationContext
                .getSharedPreferences("daily_user", Context.MODE_PRIVATE).edit();
        return editor;
    }

    public static SharedPreferences getSP(){
        SharedPreferences sharedPreferences=MyApplication.ApplicationContext
                .getSharedPreferences("daily_user",Context.MODE_PRIVATE);
        return sharedPreferences;
    }
    public static  SharedPreferences.Editor getThemeEditor(){
        SharedPreferences.Editor editor= MyApplication.ApplicationContext
                .getSharedPreferences("daily_theme", Context.MODE_PRIVATE).edit();
        return editor;
    }
    public static SharedPreferences getThemeSP(){
        SharedPreferences sharedPreferences=MyApplication.ApplicationContext
                .getSharedPreferences("daily_theme",Context.MODE_PRIVATE);
        return sharedPreferences;
    }
}

File: app/src/main/java/com/zhihudailytest/Utils/SPUtils.kt
package com.zhihudailytest.Utils

import android.content.Context
import android.content.SharedPreferences

import com.zhihudailytest.MyApplication

/**
 * Created by Administrator on 2016/8/2.
 */
object SPUtils {
    val editor: SharedPreferences.Editor
        get() = MyApplication.ApplicationContext?.getSharedPreferences("daily_user", Context.MODE_PRIVATE)!!.edit()

    val sp: SharedPreferences
        get() = MyApplication.ApplicationContext?.getSharedPreferences("daily_user", Context.MODE_PRIVATE)!!
    val themeEditor: SharedPreferences.Editor
        get() = MyApplication.ApplicationContext?.getSharedPreferences("daily_theme", Context.MODE_PRIVATE)!!.edit()
    val themeSP: SharedPreferences
        get() = MyApplication.ApplicationContext?.getSharedPreferences("daily_theme", Context.MODE_PRIVATE)!!
}
